packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -23,49 +23,88 @@ import './page.css';
 
 ## Examples
 
-### Vertical nav
+### Basic page components
+
+A page will typically contain the following components: 
+
+- A `<Page>` with a `header` that often contains a masthead, a `<PageSidebar>`, and one or more `<PageSection>` components. 
+-[Mastheads](/components/masthead) contain the `<PageToggleButton>`, a `<MastheadMain>` that contains a `<MastheadBrand>`, and the page's header toolbar within `<MastheadContent>`. 
+
+### Vertical navigation
+
+To add vertical side navigation to a `<Page>`, pass a `<PageSidebar>` component into the `sidebar` property. To render navigation in the `<PageSidebar>`, use the `nav` property. 
+
+The `isNavOpen` property helps facilitate the opening and closing of the sidebar and should be 'true' when the navigation sidebar is visible. 
 
 ```ts file="./PageVerticalNav.tsx"
 ```
 
-### Horizontal nav
+### Legacy vertical navigation
```

Comment:
As far as I can tell, this is just an example of using `<PageHeader>` instead of `<Masthead>`, which shouldn't impact vertical navigation

```suggestion
### Legacy page header
```